src/util/pg.py

```diff
@@ -996,6 +996,11 @@ def __init__(self, list_=(), quoted=()):
         self._trailing_comma = False
         self._alias = None
 
+    @classmethod
+    def _new(cls, cr, list_=()):
```

Comment:
If we need an empty one we can always use the default `ColumnList()`. We could add some docs already :)
```suggestion
    def from_unquoted(cls, cr, list_):
        """
        Build a ColumnList item from a list of column names that may need quoting.
        
        ...
        """
```
🚲 🪛  on the name :D